model/ItemsScannerService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace oat\qtiItemPci\model;
+
+use oat\taoQtiItem\model\portableElement\exception\PortableElementException;
+use oat\taoQtiItem\model\qti\interaction\PortableCustomInteraction;
+use oat\taoQtiItem\model\qti\Service as QtiService;
+
+class ItemsScannerService
+{
+    /**
+     * @var QtiService
+     */
+    private $qtiService;
+
+    /**
+     * ItemsScannerService constructor.
+     * @param QtiService $qtiServiceSingleton
+     */
+    public function __construct(QtiService $qtiServiceSingleton)
+    {
+        $this->qtiService = $qtiServiceSingleton;
+    }
+
+    /**
+     * @param string $requestTypeIndentifier
+     * @param array $allItems
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     */
+    public function isPciUsedInItems(string $requestTypeIndentifier, array $allItems)
```

Comment:
The scalar type hint (`string`) is only available in PHP7. Since we _unfortunatly_ still need to support PHP 5.6 this should be omitted.